drltrace/drltrace.c

```diff
@@ -161,8 +173,9 @@ lib_entry(void *wrapcxt, INOUT void **user_data)
             dr_fprintf(outf, "<invalid memory>");
             /* Just keep going */
         });
-        dr_fprintf(outf, ") thread id: "TIDFMT" and return to "PFX"",
-                   dr_get_thread_id(drcontext), drwrap_get_retaddr(wrapcxt));
+        dr_fprintf(outf,
+                   (options.print_ret_addr == true) ? ") and return to "PFX: ")",
```

Comment:
IMHO there should be a XXX comment about making the raw return address usable by providing module bounds